app/repo/pg/user.go

```diff
@@ -0,0 +1,29 @@
+// Package repo represents adapter layer
+// which enables interaction through a specific port
+// and with a certain technology.
+//
+// in this case repo will act
+// for CRUD operations with postgres.
+package pg
+
+import (
+	"database/sql"
+
+	"github.com/go-devs-ua/octagon/app/ent"
+	"github.com/go-devs-ua/octagon/cfg"
+)
+
+type Repo struct{ *sql.DB }
+
+func NewRepo(opt *cfg.Options) *Repo {
```

Comment:
The same thing about pointers.
Let's try, by default, to not use them. 
Once you realize that you really need it - you will add it.

Reply:
Deal. So far, it seems that I am using * mechanically